catalogue/models.py

```diff
@@ -1,3 +1,18 @@
+from django.contrib.postgres.fields import ArrayField
 from django.db import models
 
+
 # Create your models here.
+class Category(models.Model):
+    category_name = models.CharField(max_length=30)
+    category_description = models.CharField(max_length=30)
+    parent_category_id = models.ForeignKey("self", on_delete=models.CASCADE)
```

Comment:
```suggestion
    parent_category = models.ForeignKey("self", on_delete=models.CASCADE)
```
FK is OBJECT not integer field